recipe/meta.yaml

```diff
@@ -23,32 +22,28 @@ requirements:
   host:
     - python
     - pip
-    # 2022/7/14: The issue was observed with 'import PIL.Image' in v9.2.0:
-    # Additional double quotes for Core version: "9.2.0" on win-64 for python 3.7 and python 3.8.
-    # The last version of pillow we built (v9.0.1) was built using conda-build-3.21.7, 
-    # Looking at the conda-build changelog for v3.21.9, it was noticed some changes to support the new setuptools >=60.0
-    # 2023/2/23: It persists in 9.4.0. Check if the issue persists in the next releases 
-    - setuptools <60  # [win and (py==37 or py==38)]
-    - setuptools      # [not (win and (py==37 or py==38))]
+    - setuptools
     - wheel
     # Required by default
     - jpeg 9e
     - zlib 1.2.13
     # Optional dependencies
-    # (also can be incuded libwebp, libimagequant, libraqm, 
+    # (also can be incuded libwebp, libimagequant, libraqm,
     # libxcb, openjpeg (it causes build errors on win32),
     # see https://pillow.readthedocs.io/en/latest/installation.html#external-libraries)
     - freetype 2.10.4
     - lcms2 2.12       # [not win]
     - libtiff 4.2.0
     - libwebp 1.3.2
+    - openjpeg
```

Comment:
```suggestion
    - openjpeg 2.3.0
```

Reply:
It's already in the global `cbc.yaml`

Reply:
Ah ok, isn't then the pinning done in this way: `openjpeg {{openjpeg}}` ? or is it implicit?

Reply:
Implicit. I prefer to use explicit for numpy, openssl (previously for 1.x and 3.x) if there is a matrix of versions

Reply:
Ok thank you!